packages/gestalt/src/Video/Playhead.js

```diff
@@ -93,15 +99,16 @@ export default class VideoPlayhead extends PureComponent<Props, State> {
           role="progressbar"
           tabIndex="-1"
         >
-          <Box left right position="absolute" color="secondary" rounding={2} height={4}>
-            <Box color="light" rounding={2} height="100%" width={width} />
-          </Box>
           <Box
+            left
+            right
             position="absolute"
             rounding={2}
             height={4}
-            dangerouslySetInlineStyle={{ __style: { left: width } }}
```

Comment:
cn you guide me through the new logic. We removed `__style: { left` how are we making it move now? It's just `width` rn

Reply:
Got it, it's positioned to the right on rtl... so it grows that way! got it